handlers/sse.go

```diff
@@ -50,6 +69,24 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		h.eventBus.Unsubscribe(sub)
 
+		// unregister sse connection
+		numberOfConnections, err := h.UserCollection.RemoveSSEConnection(context.Background(), userID)
+		if err != nil {
+			log.Errorf("%v: %v", msg, err)
+		}
+		// desynchronize user if no more connections are active
+		if numberOfConnections == 0 {
+			_, err := h.UserCollection.SetSynchronized(context.Background(), userID, false)
```

Comment:
use `ctx` from the http request for this

Reply:
The context is expired, since the code only reaches this point after `<- ctx.Done()`